src/main/scala/is/hail/utils/ExportType.scala

```diff
@@ -0,0 +1,18 @@
+package is.hail.utils
+
+object ExportType {
+  val CONCATENATED = 0
+  val PARALLEL_SEPARATE_HEADER = 1
+  val PARALLEL_HEADER_IN_SHARD = 2
+
+  def getExportType(typ: Option[String]): Int = {
+    typ match {
+      case None => CONCATENATED
```

Comment:
I think this should take a `String` with `null` for missingness, since it's used from Python.
you can also use only one match:
```
case null => Concatenated
case Some("separate_header") => ...
```